apps/navstar_dns/src/navstar_dns_key_mgr.erl

```diff
@@ -230,6 +234,23 @@ do_create_zk_key(Pid) ->
             false
     end.
 
+encode(#{public := Pk, secret := Sk}) ->
+    {Encoding, 0} = unicode:bom_to_encoding(Pk),
```

Comment:
unicode:bom_to_encoding can return a tuple, which I don't think can be encoded as JSON